packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -202,19 +202,19 @@ export const getCanChat = createSelector(
       }
     }
 
-    const currentUserPermissions = chatPermissions[currentUserId]
-
     const userPermissions = chatPermissions[userId]
     const isBlockee = blockees.includes(userId)
     const isBlocker = blockers.includes(userId)
     const canChat =
       !isBlockee &&
       !isBlocker &&
-      (userPermissions?.current_user_has_permission ?? true)
```

Comment:
noting here that we default to user is not permitted until we load in the permissions

Reply:
yeah.. added a CTA that's "WAIT" so we can handle that case, but yeah it'll quickly flip from not allowed to allowed